docs/reference/content/driver/reference/fle.md

```diff
@@ -0,0 +1,166 @@
++++
```

Comment:
I think this belongs in the tutorial folder rather than reference, given the current split of content between the two.

Reply:
Done